SECURITY.md

```diff
@@ -0,0 +1,58 @@
+# Security Policy
+
+## Reporting a Vulnerability
+
+We take the security of Boston Open Dev's website and community infrastructure seriously. If you discover a security vulnerability, please follow these steps:
+
+1. **DO NOT** create a public GitHub issue
+2. Report via one of these secure channels (in order of preference):
+   - Send a private message on [Zulip](https://osdc.zulipchat.com/#narrow/stream/406743-boston) to the maintainers
+   - Use GitHub's private security advisory reporting
+   - Direct message on [Mastodon](https://floss.social/@bostonopen)
+3. Provide as much information as possible about:
+   - The location and nature of the vulnerability
+   - Steps to reproduce (if applicable)
+   - Potential impact of the vulnerability
+
+## What to Report
+
+Please report issues related to:
+
+- Cross-site scripting (XSS) vulnerabilities
+- Exposed sensitive information
+- Issues with authentication (if implemented)
+- Problems with data handling
+- Concerns about personal information exposure
+- Any other security-related issues
+
+## Scope
+
+In scope:
+- The bostonopen.dev website
+- Any subdomains under bostonopen.dev
+- Repository content
+- Community-related infrastructure
+
+Out of scope:
+- Third-party services we link to
+- Individual members' personal websites
+- Hypothetical attacks
+- Social engineering attempts
```

Comment:
Why is this less important?

Reply:
Great question! It's not that social engineering attempts are less important—they're just out of scope for this particular security policy because they’re generally more about human behavior and organizational practices than technical vulnerabilities in our project. That said, if you think it’s worth addressing in some form, I’m happy to revisit this and see how we can include it.